server/views/case-list.njk

```diff
@@ -157,10 +157,14 @@
 
     {% if params.unmatchedRecords > 0 %}
         {%- from "components/inset-text/macro.njk" import insetText -%}
+        {% set defendantCount = 'defendant' if params.unmatchedRecords <= 1 else 'defendants' %}
```

Comment:
`defendantPluralised` might be more descriptive than `Count` for this and the 3 variables below

Reply:
👍